### PR TITLE
Added IP_Address test cases

### LINKED ISSUE #17: IP_ADDRESS:  Internet Protocol (IP) address (either IPv4 or IPv6).
### Description
When you surf the web and go online you have something called a IP Address, this allows you to differentiate while on the internet. You need this to search the web or scroll online to learn more about it [here](https://www.fortinet.com/resources/cyberglossary/what-is-ip-address#:~:text=What%20is%20the%20use%20of,%2Darea%20networks%20(LANs).)

  - Why is it important? 
    In the world we live in today almost everything can be done online, such as shopping, checking your bank account or even paying bills. So if a hacker were to come in and do something like a man-in-the-middle attack, they could get access to your sensitive information. Then they could possibility pretend to be you and steal money or sell your information to others.
     
### Detection Steps
For detecting an IPv4 address

  - all ips are start with 3 numbers followed by a period such as 216.27.61.137
  - they use 32 binary bits to create a single unique address

to learn more click either of this link [one ](https://computer.howstuffworks.com/internet/basics/what-is-an-ip-address.htm)

### PR DESCRIPTION
**Pull request by Pojohnson-XJ**

### CODE REVIEW

**Review by ZachLCOMP — APPROVED** [review]
Looks pretty good to me. All tests are passing

File: test_team_1.py [hunk 1]
```diff
@@ -89,6 +89,42 @@ def test_iban_code(self):
 
     def test_ip_address(self):
         """Test IP_ADDRESS functionality"""
+        net1s = ['168', '192', '255'] #First part of the IP Address
+        net2 = ['243', '542', '495']  #Second part of the IP Address
```
Comment (claesmk) [inline test_team_1.py]: 542 and 495 are not valid octets

File: test_team_1.py [hunk 2]
```diff
@@ -89,6 +89,42 @@ def test_iban_code(self):
...
+            for n2 in net2:
+                for n3 in net3:
+                    for h in hostID:
+                        IPAddres = '.'.join([n1,n2,n3,h])
+                        result = analyze_text('IP_ADDRESS is ' + IPAddres, ['IP_ADDRESS']) #One with Context words
+                        result2 = analyze_text('my Address is ' + IPAddres, ['IP_ADDRESS'])#One without context words
+                        print('\n',IPAddres)
+                        print(result, '\n')
+                        print(result2)
+                        break
```
Comment (claesmk) [inline test_team_1.py:114]: You are not actually testing anyting here, only printing the results.  You need an appropriate self.assert() statement to actually test something.

File: test_team_1.py [hunk 3]
```diff
@@ -89,6 +89,42 @@ def test_iban_code(self):
...
+        for IP1 in IP61:
+            for IP2 in IP62:
+                IP6 = ':'.join([IP1,IP2])
+                result = analyze_text('IP_ADDRESS is ' + IP6, ['IP_ADDRESS'])#One with Context words
+                result2 = analyze_text('my Road is ' + IP6, ['IP_ADDRESS'])#One without context words
+                print('\n',IP6)
+                print(result, '\n')
+                print(result2)
```
Comment (claesmk) [inline test_team_1.py:124]: Similar comment here - needs a valid self.assert() statement

**Review by claesmk — CHANGES_REQUESTED** [review]
Please update the positive test cases as noted

**Review by claesmk — APPROVED** [review]
Looks OK now

**Pull request merged**